connectors/spark/src/test/java/io/unitycatalog/connectors/spark/TableReadWriteTest.java

```diff
@@ -41,6 +41,33 @@ public class TableReadWriteTest extends BaseSparkIntegrationTest {
 
   private TableOperations tableOperations;
 
+  @Test
+  public void testNoDeltaCatalog() throws IOException, ApiException {
+    SparkSession.Builder builder = SparkSession.builder()
+            .appName("test")
+            .master("local[*]")
+            .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension");
+    String catalogConf = "spark.sql.catalog.spark_catalog";
+    builder = builder
+            .config(catalogConf, UCSingleCatalog.class.getName())
+            .config(catalogConf + ".uri", serverConfig.getServerUrl())
+            .config(catalogConf + ".token", serverConfig.getAuthToken())
+            .config(catalogConf + ".__TEST_NO_DELTA__", "true");
+    SparkSession session = builder.getOrCreate();
+
+    setupExternalParquetTable(PARQUET_TABLE, new ArrayList<>(0));
+    testTableReadWrite(SPARK_CATALOG + "." + SCHEMA_NAME + "." + PARQUET_TABLE, session);
+
+    // Since `DeltaCatalog` is not used, the following CREATE TABLE command doesn't fail while it
+    // should because the table location is empty and Delta Lake shouldn't allow it.
```

Comment:
i dont get this. why should this pass?? if delta library is not in the classpath, then all queries with the format delta should fail

Reply:
we cannot really test that in these unit tests since delta is in the class path. so ... what is this test really testing?

Reply:
`DeltaCatalog#createTable` errors out if the table to be created is external but the location is empty. This is to prove that `DeltaCatlaog` is not used in the CREATE TABLE workflow. If I remove `.config(catalogConf + ".__TEST_NO_DELTA__", "true")` in this test case, this command will fail